Packages/com.vrchat.ClientSim/Runtime/CameraStacking/ClientSimStackedVRCameraSystem.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.XR;
+using VRC.Core;
+using VRC.SDK3.ClientSim;
+
+namespace VRC.SDK3.ClientSim
+{
+    public class ClientSimStackedVRCameraSystem : MonoBehaviour
+    {
+        public ClientSimStackedCamera[] CameraStack;
+        [HideInInspector] public bool IsCameraStackingEnabled = false;
+        
+        private Camera MainSceneCamera;
+        private bool IsInitialized = false;
+        private bool IsReady = false;
+        private List<Camera> Cameras;
+        
+        private ClientSimMenu clientSimMenu;
+
+        public void Initialize(Camera playerCamera, ClientSimMenu menu)
+        {
+            MainSceneCamera = playerCamera;
+            clientSimMenu = menu;
+        }
+
+        public void Ready()
+        {
+            IsReady = true;
+        }
+        
+        void Update()
+        {
+            if(!IsReady) return;
+            if (!IsInitialized) { InitializeStackedSystem(); }
+            else
+            {
+                if (!Cameras[0].enabled)
+                {
+                    Debug.LogError("Stacked Cameras are not enabled.");
+                }
+            }
+        }
+     
+        /*void OnEnable()
+        {
+            #if !UNITY_ANDROID
+            if (MainSceneCamera != null)
+            {
+                if (!IsCameraStackingEnabled)
+                    CreateCameraStack();
+            }
+            #endif
+        }
+*/
```

Comment:
Hm, what's this about?

Reply:
oops! it's not needed for the way we initialize camerastacking in clientsim, forgot to delete after I got it working